ee/secureenclavesigner/secureenclavesigner.go

```diff
@@ -0,0 +1,197 @@
+//go:build darwin
+// +build darwin
+
+package secureenclavesigner
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/kolide/krypto/pkg/echelper"
+	"github.com/vmihailenco/msgpack/v5"
+)
+
+type SignRequest struct {
```

Comment:
Kind a weird that 2 / 3 members are expected to be []byte(B64) ... kind a wonder if they should all be B64 strings or all be expect to be in straight []bytes (without B64)